packages/flutter_tools/templates/plugin/.gitignore.tmpl

```diff
@@ -1,10 +1,11 @@
 .DS_Store
-.atom/
 .dart_tool/
-.idea
+
 .packages
 .pub/
+pubspec.lock
+
 build/
 ios/.generated/
-packages
-pubspec.lock
+ios/Flutter/Generated.xcconfig
```

Comment:
prefer adding them to the ios.tmpl subdirectory

Reply:
Done.